nodegit.nix

```diff
@@ -0,0 +1,45 @@
+{ pkgs }:
+
+let
+  version = "0.20.3";
+
+  stdenv = pkgs.stdenv;
+  fetchurl = pkgs.fetchurl;
+
+  curl = pkgs.curl.override {
+    sslSupport = false;
+    gnutlsSupport = true;
+  };
+  glibc = pkgs.glibc;
+  openssl = pkgs.openssl;
+
+  libs = [ stdenv.cc.cc curl glibc openssl ];
+
+in
+
+  assert pkgs.stdenv.system == "x86_64-linux";
+
+  stdenv.mkDerivation {
+    name = "nodegit-${version}";
+
+    src = fetchurl {
+      url = "https://nodegit.s3.amazonaws.com/nodegit/nodegit/nodegit-v${version}-node-v51-linux-x64.tar.gz";
+      sha256 = "d022a88e58fa70f78b59b475fa6ecc6724d372a779b132264cf571f71bc50020";
+    };
+
+    buildPhase = ":";
```

Comment:
What about that :) ?

Reply:
To be honest I've blindly followed https://nixos.wiki/wiki/Packaging/Binaries#Creating_the_Derivation_for_upstream_Packaging